src/trajectory_data_handling/sqlite_querying.cpp

```diff
@@ -9,6 +9,8 @@ namespace trajectory_data_handling {
     int query_handler::m_order{};
     std::shared_ptr<std::vector<data_structures::Trajectory>> trajectory_data_handling::query_handler::original_trajectories;
     std::shared_ptr<std::vector<data_structures::Trajectory>> trajectory_data_handling::query_handler::simplified_trajectories;
+    std::vector<std::string> query_handler::id{};
```

Comment:
Kan i ikke tilføje disse members til klassen i hpp filen, når de ikke har nogen kompleks initialization? Det handler om alle members på linje 6-12

Reply:
Det samme gælder for linje 16-18